codec-http2/src/main/java/io/netty/handler/codec/http2/Http2ConnectionHandler.java

```diff
@@ -167,6 +165,10 @@ public Http2ConnectionEncoder encoder() {
         return encoder;
     }
 
+    public void frameListener(Http2FrameListener listener) {
```

Comment:
Maybe not have this at all?  If the handler wants to listen, it's easy enough to do `decoder().frameListener(this)`. WDYT?